opentech/urls.py

```diff
@@ -25,6 +25,7 @@
     url('^sitemap\.xml$', sitemap),
 
     url(r'^apply/', include(apply_urls)),
+    url(r'^user/', include(users_urls))
```

Comment:
Should this be `account` instead of `user`. `user` seems developer focused.

Reply:
let's discuss

Reply:
Raised with Dan.